### DISCUSSION TITLE
Weather RTL language support

### DISCUSSION THREAD

**Discussion by abd3lraouf** [valinet/ExplorerPatcher#954] [Q&A] [answered]
I have the following issue when using the Arabic language `ar`
![2022-02-28 08_33_51-Alienware Command Center](https://user-images.githubusercontent.com/11881564/155935609-abbf91e6-fe03-4c58-bb7c-190f02df11e2.png)
![2022-02-28 08_34_05-Mail](https://user-images.githubusercontent.com/11881564/155935613-086dd06e-06de-47d2-81ec-8198a217ee7d.png)

**Comment by valinet** [comment]
Yeah, expected that… hmm, let me think of a solution and I will come back with an update.

**Comment by valinet** [comment, marked as answer]
I fixed this in the latest pre-release (42.22).

**Reply by abd3lraouf** [reply]
Thank you I can confirm it's now fixed 😍🤗
